{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -1,21 +1,14 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-"""
-test_{{ cookiecutter.project_slug }}
-----------------------------------
-
-Tests for `{{ cookiecutter.project_slug }}` module.
-"""
+"""Tests for `{{ cookiecutter.project_slug }}.{{ cookiecutter.project_slug }}` module."""
```

Comment:
This is a very minor nitpicking, but considering this file may also contain tests for the `{{ cookiecutter.project_slug }}.cli` , maybe it's better to leave only ``{{ cookiecutter.project_slug }}` ? :)